jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java

```diff
@@ -804,8 +805,10 @@ public String toString()
     @Override
     public void dump(Appendable out, String indent) throws IOException
     {
-        dumpObjects(out, indent, new org.eclipse.jetty.util.component.ClassLoaderDump(this.getClass().getClassLoader()),
+        dumpObjects(out, indent,
+            new org.eclipse.jetty.util.component.ClassLoaderDump(this.getClass().getClassLoader()),
```

Comment:
Remove the deprecated `.server.ClassLoaderDump` class entirely, so you have to use full qualified names here.